README.CentOS

```diff
@@ -0,0 +1,43 @@
+## Quickstart on CentOS7 ##
+
+
+Note: This assumes an Infrastructure Server install for x64. The additional 
+dependency packages that need to be installed may vary if you are starting with
+a different base system.
+
+
+Install Dependencies:
+
+sudo yum update
+sudo yum install epel-release
+sudo yum install cmake gcc-c++ libevent-devel golang autoconf pkgconfig \
+    json-c-devel gflags-devel glog-devel protobuf-devel leveldb-devel \
+    openssl-devel gperftools-devel protobuf-compiler sqlite-devel ant \
+    java-1.8.0-openjdk-devel protobuf-java python-gflags protobuf-python \
+    python-ecdsa python-mock python-httplib2 git ldns-devel automake \
```

Comment:
You shouldn't need the following since they're part of the gclient build:
* libevent-devel 
* json-c-devel 
* gflags-devel
* glog-devel
* protobuf-devel
* leveldb-devel
* openssl-devel
* gperftools-devel
* protobuf-compiler
* sqlite-devel
* ldns-devel
* libunwind-devel
```